web_src/js/components/DashboardRepoList.vue

```diff
@@ -121,27 +119,27 @@
           {{ textMyOrgs }}
           <span class="ui grey label gt-ml-3">{{ organizationsTotalCount }}</span>
         </div>
-        <a v-if="canCreateOrganization" :href="subUrl + '/org/create'" :data-tooltip-content="textNewOrg">
+        <a class="gt-df gt-ac muted" v-if="canCreateOrganization" :href="subUrl + '/org/create'" :data-tooltip-content="textNewOrg">
           <svg-icon name="octicon-plus"/>
           <span class="sr-only">{{ textNewOrg }}</span>
         </a>
       </h4>
       <div v-if="organizations.length" class="ui attached table segment gt-rounded-bottom">
         <ul class="repo-owner-name-list">
-          <li v-for="org in organizations" :key="org.name">
-            <a class="repo-list-link gt-df gt-ac gt-sb" :href="subUrl + '/' + encodeURIComponent(org.name)">
-              <div class="text truncate item-name gt-f1">
-                <svg-icon name="octicon-organization" :size="16" class-name="gt-mr-2"/>
-                <strong>{{ org.name }}</strong>
-                <span class="ui tiny basic label gt-ml-3" v-if="org.org_visibility !== 'public'">
+          <li class="gt-df gt-ac" v-for="org in organizations" :key="org.name">
+            <a class="repo-list-link muted gt-df gt-ac gt-f1" :href="subUrl + '/' + encodeURIComponent(org.name)">
+              <svg-icon name="octicon-organization" :size="16" class-name="repo-list-icon"/>
+              <div class="text truncate">{{ org.name }}</div>
+              <div>
```

Comment:
Does it really need this empty `<div>` wrapper?

It seems that there is only one `<span>` inside. Is this `<div>` a no-op now?

Reply:
Whithout the div, this happens:

<img width="140" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/eae1b917-be56-44e5-8300-a1d38d420d04">

Don't ask me how the presence of the div prevents the underline, but it does.

Reply:
Added a comment. I see no other solution. Decoration effects are like this can not be effectively prevented/undone via CSS.